ArgoProject/src/factories/scenes/SplashSceneFactory.cpp

```diff
@@ -13,6 +14,7 @@ std::vector<app::Entity> app::fact::sce::SplashSceneFactory::create()
 	auto entities = std::vector<app::Entity>();
 
 	auto player = fact::PlayerFactory().create();
+	auto level = fact::LevelFactory().create();
```

Comment:
level factory returns a vector of entities, please insert those entities into the above vector.
```suggestion
	auto level = fact::LevelFactory().create();
	entities.insert(entities.end(), level.begin(), level.end());
```